### PR TITLE
Add a new data product

### PR DESCRIPTION
**Pull request by TheHelias**
Added a data product for stradler machines
image

### CODE REVIEW

File: demo_data/images/straddle-carrier.jpg
Comment (lietu) [inline demo_data/images/straddle-carrier.jpg:1]: Please crop to square + scale down to 512x512, or ask whoever gave you this to do that

Comment (joakimnordling) [reply demo_data/images/straddle-carrier.jpg:1]: Well, now it's just 80x80, not 512x512.

File: scanner/src/lib/dataproducts/MobileWorkMachine_StraddleCarrier_OperationsData_v0.1.svelte [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  /*
+        Docs:
+
+        https://definitions.sandbox.ioxio-dataspace.com/definitions/DigitalProductPassport/MobileWorkMachine/StraddleCarrier/OperationsData_v0.1
+
+        https://github.com/ioxio-dataspace/sandbox-definitions/blob/main/src/DigitalProductPassport/MobileWorkMachine/StraddleCarrier/OperationsData_v0.1.py
+
+       https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_post
```
Comment (joakimnordling) [inline scanner/src/lib/dataproducts/MobileWorkMachine_StraddleCarrier_OperationsData_v0.1.svelte:9]: Indentation is off for the last entry compared to the above ones. Seems like these comments are in any case not uniform in these dataproducts/*.svelete files when it comes to indentation but would be kind of nice if they at least within each file would be.

Comment (joakimnordling) [reply scanner/src/lib/dataproducts/MobileWorkMachine_StraddleCarrier_OperationsData_v0.1.svelte:9]:
```suggestion
        https://gateway.sandbox.ioxio-dataspace.com/docs#/Data%20Products/DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_DigitalProductPassport_MobileWorkMachine_StraddleCarrier_OperationsData_v0_1_post
```

**Review by joakimnordling — APPROVED** [review]

**Review by lietu — APPROVED** [review]

**Pull request merged**